hugegraph-dist/src/assembly/static/conf/hugegraph.properties

```diff
@@ -5,9 +5,11 @@ gremlin.graph=com.baidu.hugegraph.HugeFactory
 # cache config
 #schema.cache_capacity=100000
 # vertex-cache default is 1000w, 10min expired
+vertex.cache_type=L2
```

Comment:
expect l2(lowercase,  not L2)

Reply:
ok，code has been updated and resubmitted